sql/core/src/main/scala/org/apache/spark/sql/execution/ui/SparkPlanGraph.scala

```diff
@@ -101,12 +104,16 @@ private[sql] object SparkPlanGraph {
         } else {
           subgraph.nodes += node
         }
+        // ShuffleExchange or BroadcastExchange
```

Comment:
Is there a more robust way to do this? If not, can you move "Exchange" to the interface as a constant so if we add more, we know they need to be named this way?